2d/platformer/src/titlescreen/TitleScreen.gd

```diff
@@ -0,0 +1,20 @@
+extends Node
+
+
+
+func _ready():
+	$MarginContainer/VBoxContainer/VBoxContainer/TextureButton.grab_focus()
+
+func _physics_process(delta):
+	if $MarginContainer/VBoxContainer/VBoxContainer/TextureButton.is_hovered() == true:
+		$MarginContainer/VBoxContainer/VBoxContainer/TextureButton.grab_focus()
+	if $MarginContainer/VBoxContainer/VBoxContainer/TextureButton2.is_hovered() == true:
+		$MarginContainer/VBoxContainer/VBoxContainer/TextureButton2.grab_focus()
+
+
+func _on_TextureButton_pressed():
+	get_tree().change_scene("res://src/Main/Game.tscn")
+
+
+func _on_TextureButton2_pressed():
```

Comment:
These two methods should be renamed to reflect the change in button name.

Reply:
I see how this may work, given their functionality. However I agree with Reid. based on the names of the functions alone, I cannot tell what TextureButton  or TextureButton2 is supposed to be in general.

Reply:
Unable to change function names. Stops buttons from working.

Reply:
Have you tried updating the signals as well? Lines 65 and 66 in TitleScreen.tscn. You _should_ be able to name then anything you want (the method and button)